lib/child_process.js

```diff
@@ -347,7 +347,11 @@ function normalizeSpawnArguments(file /*, args, options*/) {
     }
   }
 
-  args.unshift(file);
+  if ('name' in options) {
```

Comment:
Perhaps this should be a bit more specific, like `typeof options.name === 'string'`.